dvc/dvcfile.py

```diff
@@ -66,7 +67,19 @@ def is_lock_file(path):
     return os.path.basename(path) == PIPELINE_LOCK
 
 
-def check_dvc_filename(path):
+def is_git_ignored(repo, path):
+    from dvc.fs.local import LocalFileSystem
+    from dvc.scm.base import NoSCMError
+
+    try:
+        return isinstance(repo.fs, LocalFileSystem) and repo.scm.is_ignored(
+            path
+        )
+    except NoSCMError:
+        return False
+
+
+def check_dvc_filename(repo, path):
```

Comment:
Shouldn't we rename it? It is no longer about filename. Maybe `check_dvc_file` or `check_dvc_file_path`?